src/components/WebEditor/tabs.jsx

```diff
@@ -76,7 +78,7 @@ function a11yProps(index) {
 
 export default function BasicTabs() {
   const [value, setValue] = React.useState(0);
-  const { setJs, problems, setExp } = React.useContext(DataContext);
+  const { setJs, problems, setExp, js } = React.useContext(DataContext); // added js to the destructured variables
```

Comment:
I am assuming DataContext was already exporting js and nothing has been changed on DataContext. Please confirm

Reply:
Yes, that is correct
`    <DataContext.Provider
      value={{
        js,
        setJs,
        problems,
        setProblems,
        exp,
        setExp
      }}
    >
      {children}
    </DataContext.Provider>`